app/src/renderer/components/staking/PageValidator.vue

```diff
@@ -1,89 +1,129 @@
-        tm-list-item(dt="Website" :dd="validator.description.website")
-        tm-list-item(dt="Details" :dd="validator.description.details")
-
-    tm-part(title='Validator Keys')
-      tm-list-item(dt="Owner" :dd="validator.owner")
-      tm-list-item(dt="Pub Key" :dd="validator.pub_key")
-
-    tm-part(title='Validator Stake' v-if="!validator.revoked")
-      tm-list-item(dt="Voting Power" :dd="pretty(validator.voting_power) + ' ' + this.config.bondingDenom")
-      tm-list-item(dt="Self Bonded" :dd="pretty(selfBond) + ' ' + this.config.bondingDenom")
-      tm-list-item(dt="Bond Height" :dd="`Block ${validator.bond_height}`")
-
-    tm-part(title='Commission')
-      tm-list-item(dt="Commission" :dd="pretty(validator.commission) + ' %'")
-      tm-list-item(dt="Commission Maximum" :dd="pretty(validator.commission_max) + ' %'")
-      tm-list-item(dt="Commission Change-Rate" :dd="pretty(validator.commission_change_rate) + ' %'")
-      tm-list-item(dt="Commission Change Today" :dd="pretty(validator.commission_change_today) + ' %'")
+    .validator-profile__header.validator-profile__section
+      .column
+        img.avatar(v-if="validator.keybase" :src="validator.keybase.avatarUrl")
+        img.avatar(v-else src="~assets/images/validator-icon.svg")
+      .column.validator-profile__header__info
+        .row.validator-profile__header__name
+          .column
+            div.validator-profile__status-and-title
+              span.validator-profile__status(v-bind:class="statusColor" v-tooltip.top="status")
+              .validator-profile__header__name__title {{ validatorTitle(validator) }}
+            //- TODO replace with address component when ready
+            anchor-copy.validator-profile__header__name__address(:value="validator.owner" :label="shortAddress(validator.owner)")
+          .column.validator-profile__header__actions
+            tm-btn(id="stake-btn" value="Stake" color="primary" @click.native="onStake()")
+            tm-btn(v-if="config.devMode" value="Unstake" color="secondary")
+        .row.validator-profile__header__data
+          dl.colored_dl
+            dt My Stake
+            dd {{myBond < 0.01 ? '< ' + 0.01 : pretty(myBond)}}
+          dl.colored_dl(v-if="config.devMode")
+            dt My Rewards
+            dd n/a
+          .validator-profile__header__data__break
+          dl.colored_dl
+            dt Voting Power
+            dd(id="validator-profile__power" v-bind:class="[powerRatioLevel]") {{pretty(powerRatio * 100)}} %
+          dl.colored_dl(v-if="config.devMode")
+            dt Uptime
+            dd n/a
+          dl.colored_dl
+            dt Commission
+            dd(id="validator-profile__commission" v-bind:class="[commissionLevel]") {{pretty(validator.commission)}} %
+          dl.colored_dl(v-if="config.devMode")
+            dt Slashes
+            dd n/a
+
+    .validator-profile__details.validator-profile__section
+      .row
+        .column
+          dl.info_dl
```

Comment:
maybe for another PR/issue, but can we change the box background color to gray? It looks like an input field rn... (cc: @jbibla)

Reply:
let's merge it for now, i will keep iterating

Reply:
@fedekunze can you create an issue for this please?

Reply:
See #1324